### PR TITLE
Scaffold info

### PR DESCRIPTION
**Pull request by mixcas**
**Must to create `info` page**

### CODE REVIEW

File: page-info.php [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+            <ul>
+
+      <?php
+      foreach ($team as $teammate) {
+        $teammate_picture = wp_get_attachment_image($teammate['_igv_teammate_picture_id'], 'some-size'); // TODO: Change image size
+        $teammate_name = $teammate['_igv_teammate_name'];
+        $teammate_description = apply_filters('the_content', $teammate['_igv_teammate_description']);
+      ?>
+              <li>
+                <?php echo $teammate_picture; ?>
```
Comment (varffs) [inline page-info.php]: no conditional for failure here?

File: page-info.php [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+    <?php
+    if (!empty($team)) {
+    ?>
+
+          <section>
+            <ul>
+
+      <?php
+      foreach ($team as $teammate) {
+        $teammate_picture = wp_get_attachment_image($teammate['_igv_teammate_picture_id'], 'some-size'); // TODO: Change image size
```
Comment (varffs) [inline page-info.php]: I dont think the size should be a todo. No reason not to set now.

File: page-info.php [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+
+    <?php
+    }
+    ?>
+
+    <?php
+    if (!empty($collaborators)) {
+    ?>
+          <section>
+            <h2>Clientes</h2>
```
Comment (varffs) [inline page-info.php]: Wrong title

File: page-info.php [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+    the_post();
+
+    $team = get_post_meta($post->ID, '_igv_teammate_group', true);
+    $collaborators = get_post_meta($post->ID, '_igv_collaborator_group', true);
+    $clients = get_post_meta($post->ID, '_igv_client_group', true);
+    $services = get_post_meta($post->ID, '_igv_service_group', true);
+    $contact = get_post_meta($post->ID, '_igv_contact', true);
+?>
+
+        <article <?php post_class('grid-item item-s-12'); ?> id="post-<?php the_ID(); ?>">
```
Comment (varffs) [inline page-info.php]: this should just be the main container

File: page-info.php [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+            <h2>About</h2>
+    <?php the_content(); ?>
+
+    <?php
+    if (!empty($team)) {
+    ?>
+
+          <section>
```
Comment (varffs) [inline page-info.php]: each section should be a row and if needed the inner content split into grid items

**Review by varffs — CHANGES_REQUESTED** [review]

File: page-info.php (hunk 4 above)
Comment (mixcas) [reply page-info.php]: what do u mean?

File: page-info.php (hunk 1 above)
Comment (mixcas) [reply page-info.php]: doesn't need one, for now

**Comment by mixcas** [comment]
@varffs plz check

**Review by varffs — APPROVED** [review]

**Pull request merged**